packages/node-cli/messages.json

```diff
@@ -1,12 +1,12 @@
 {
 	"ANSWERS_VALIDATION_MESSAGES_ALPHANUMERIC_HYPEN_UNDERSCORE": "This field contains forbidden characters. Use only alphanumerical characters, hyphens, or underscores.",
+	"ANSWERS_VALIDATION_MESSAGES_ALPHANUMERIC_HYPEN_UNDERSCORE_EXTENDED": "This field contains forbidden characters. Use only alphanumerical characters, hyphens, underscores, spaces, or periods. The project name cannot begin or end with a period.",
```

Comment:
Also, I prefer to write the characters instead of their names... I understand it is better literary speaking but it is clearer if we show them as it was before. I think at least windows shows them like this

Reply:
I agree with showing the actual characters in the error message.

Reply:
I agree with showing the actual characters in the error message.

Reply:
Hypen changed to Hyphen, but the message is the one agreed